source/includes/harvest/_job_posts.md

```diff
@@ -250,3 +250,98 @@ questions | If present, will return an array of questions associated with this j
 
 <br>
 [See noteworthy response attributes.](#the-job-post-object)
+
+## Retrieve multiple job posts for a job
```

Comment:
Let's say 'List job posts for a job' which is the more conventional way of saying this